AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -208,6 +209,25 @@ - (void)removePlayerItemObservers:(AVPlayerItem *)playerItem
   [notificationCenter removeObserver:self name:AVPlayerItemNewErrorLogEntryNotification object:playerItem];
 }
 
+- (void)addPlayerObservers:(AVPlayer *)player
+{
+  if (player == nil) {
+    return;
+  }
+
+  [player addObserver:self forKeyPath:kRate options:NSKeyValueObservingOptionNew context:ASVideoNodeContext];
+}
+
+- (void) removePlayerObservers:(AVPlayer *)player
+{
+  @try {
+    [player removeObserver:self forKeyPath:kRate context:ASVideoNodeContext];
+  }
+  @catch (NSException * __unused exception) {
+    NSLog(@"Unnecessary KVO removal");
+  }
```

Comment:
No it's not getting thrown. To be honest I simply followed the example of [removePlayerItemObservers](https://github.com/flovouin/AsyncDisplayKit/blob/f68132c96db12516260755926dc690bee7f673d2/AsyncDisplayKit/ASVideoNode.mm#L194), but I've never seen either of these exceptions being thrown.